src/main/webapp/app/oncokb-annotation-visualisation/AnnotationVisualisation.tsx

```diff
@@ -0,0 +1,380 @@
+import * as React from 'react';
+import { COLOR_BLUE } from './config/theme';
+import { GenePageTable } from './components/tables/GenePageTable';
+import { SearchColumn } from './components/tables/OncoKBTable';
+import { computed } from 'mobx';
+import {
+  getDefaultColumnDefinition,
+  filterByKeyword,
+} from './components/Utils';
+import {
+  AnnotationImplication,
+  TreatmentImplication,
+  NotificationImplication,
+} from './config/constants';
+import Tab from './components/tabs/Tab';
+import Tabs from './components/tabs/Tabs';
+import Notification from './components/notifications/notifications';
+import './components/styles/index.module.scss';
+import TabNumbers from './components/icons/TabNumbers';
+import {
+  defaultAnnotationColumns,
+  defaultTreatmentColumns,
+} from './config/constants';
+import { TREATMENTS_TABLE_COLUMN_KEY } from './config/constants';
+export interface AnnotationVisualisationProps {
+  annotations: AnnotationImplication[];
+  treatments: TreatmentImplication[];
+  patientId: string;
+  patientInfo: string;
+  isPatientInfoVisible?: boolean;
+  notifications: NotificationImplication[];
+}
+import { annotationColumns, treatmentColumns } from './config/constants';
+
+export interface AnnotationVisualisationState {
+  selectedAnnotationColumns: string[];
+  selectedTreatmentColumns: string[];
+}
+export class AnnotationVisualisation extends React.Component<
```

Comment:
Could you make this a function component. So just `export function AnnotationVisualisation() { ... }`